server/src/main/java/org/elasticsearch/rest/action/cat/RestAllocationAction.java

```diff
@@ -84,6 +86,11 @@ protected Table getTableWithHeader(final RestRequest request) {
         final Table table = new Table();
         table.startHeaders();
         table.addCell("shards", "alias:s;text-align:right;desc:number of shards on node");
+        table.addCell(
+            "forecast.write_load",
+            "default:false;alias:fwl,forecastWriteLoad;text-align:right;desc:sum of index write load forecasts"
+        );
+        table.addCell("forecast.shard_size", "default:false;alias:fd,forecastShardSize;text-align:right;desc:sum of shard size forecasts");
```

Comment:
I would like to include them by default to they are immediately available in diagnostics bundle.
I am currently trying to update yaml tests, they are very tricky for cat apis around huge regexps

Reply:
I think changing the default fields in a cat API would be a breaking change. We can add fields to JSON APIs but not here.

Reply:
I've been convinced otherwise in this regard, we can add these columns by default and I'd prefer to do so.